### PR TITLE
Grid cells using waltz-entity-link do not work correctly

### PR DESCRIPTION
**Pull request by rovats**
#856

### CODE REVIEW

File: waltz-ng/client/common/componets/entity-link/entity-link.js [hunk 1]
```diff
@@ -17,16 +17,25 @@ const initialState = {
...
 
 
-function controller() {
+function controller($state) {
     const vm = initialiseData(this, initialState);
 
     vm.$onChanges = (changes) => {
         if (vm.entityRef) {
             vm.viewState = kindToViewState(vm.entityRef.kind);
+            if (vm.viewState) {
```
Comment (davidwatkins73) [inline waltz-ng/client/common/componets/entity-link/entity-link.js]: do we need `viewState` bound to `vm` any more ?

Comment (rovats) [inline waltz-ng/client/common/componets/entity-link/entity-link.js]: used by ng-if

Comment (davidwatkins73) [inline waltz-ng/client/common/componets/entity-link/entity-link.js]: couldn't that use the view url?

Comment (rovats) [inline waltz-ng/client/common/componets/entity-link/entity-link.js]: but yes, I could just use `viewUrl` in the ng-if now instead.. will make the change

**Pull request merged**